lib/livebook_web/templates/error/404.html.eex

```diff
@@ -4,15 +4,16 @@
     <meta charset="utf-8"/>
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <link rel="icon" type="image/svg+xml" href="<%= Routes.static_path(@conn, "/favicon.svg") %>" />
+    <link rel="alternate icon" type="image/png" href="<%= Routes.static_path(@conn, "/favicon.png") %>" />
     <title><%= @status %> - Livebook</title>
     <link rel="stylesheet" href="<%= Routes.static_path(@conn, "/css/app.css") %>"/>
   </head>
   <body>
     <div class="h-screen flex items-center justify-center bg-gray-900">
       <div class="flex flex-col space-y-4 items-center">
-        <a href="/">
-          <img src="/images/logo.png" height="128" width="128" alt="livebook" />
+        <a href="<%= Routes.static_path(@conn, "/") %>">
```

Comment:
Please use url route helpers instead.

Reply:
Change to `Routes.path`, `Routes.url` is not defined with string as argument, is it ok?

Reply:
What I mean is something like `Routes.home_path(@conn)` or whatever is our current helper for the home page!

Reply:
Actually, the change you did is enough for now. We are going to rename those to the new verified helpers soon anyway!